fineract-core/src/main/java/org/apache/fineract/infrastructure/event/external/exception/ExternalEventConfigurationNotFoundException.java

```diff
@@ -21,7 +21,7 @@
 public class ExternalEventConfigurationNotFoundException extends RuntimeException {
 
     public ExternalEventConfigurationNotFoundException() {
-        super("All external events are not configured");
+        super("No external events configured");
```

Comment:
https://github.com/apache/fineract/blob/2b4c932ddd62a50c686be314c442aafa1bae2eee/fineract-core/src/main/java/org/apache/fineract/infrastructure/event/external/service/ExternalEventConfigurationValidationService.java#L78-L80

This exception is thrown, when number of events found on the classpath are not matching with the number of the events found in the database for the tenant. So I am not sure that the `No external events configured` is good here.

Reply:
Renaming the exception doesn't seem to have any unintended side effect... I did it, because "All ... are not" is simply just "No...". It's just a question of brevity... I think you would say "All external events are configured" for the positive case, but "No external events configured" for the negative case (i. e. this failure/exception situation). "All external events are not configured" sounds a bit weird.

Reply:
I would rather with "Not all external events are configured" because that is the closer to the truth.